packages/EasyCore/src/Bridge/Symfony/Doctrine/EntityManagerResolver.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyCore\Bridge\Symfony\Doctrine;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\Persistence\ManagerRegistry;
+
+final class EntityManagerResolver
+{
+    /**
+     * @var \Doctrine\Persistence\ManagerRegistry
+     */
+    private $managerRegistry;
+
+    public function __construct(ManagerRegistry $managerRegistry)
+    {
+        $this->managerRegistry = $managerRegistry;
+    }
+
+    /**
+     * @return \Doctrine\ORM\EntityManagerInterface
```

Comment:
That's not needed because the method has a return type hint

Reply:
Done.